docs/technical/index.rst

```diff
@@ -0,0 +1,9 @@
+====================================
+``edx-platform`` technical reference
+====================================
+
+This is the reference technical documentation for edx-platform.
```

Comment:
Nit: I think `technical documentation reference` sounds a little better, but not a big deal either way.

Reply:
:ok_hand: